source/extensions/filters/http/aws_lambda/aws_lambda_filter.cc

```diff
@@ -165,6 +165,11 @@ void AWSLambdaFilter::onSuccess(
   }
   aws_authenticator_.init(access_key, secret_key, session_token);
 
+  if (filter_config_->propagateOriginalRouting_){
```

Comment:
```suggestion
  if (filter_config_-> propagate_original_routing_){
```

Reply:
agree with this style nit, without the space